api/src/controllers/assets.ts

```diff
@@ -127,7 +127,7 @@ router.get(
 
 		const { stream, file, stat } = await service.getAsset(req.params.pk, transformation, range);
 
-		const access = !!req.accountability?.role ? 'private' : 'public';
+		const access = req.accountability?.role ? 'private' : 'public';
```

Comment:
Was this an automated change?

Reply:
Exactly! Is something wrong with this?

Reply:
No, just personal preference. I personally kinda use the `!!` as an "exists"

Reply:
Fair enough! As I won't include automated changes in my pull request, this won't be affected.
If you prefer to keep it you can turn off the [no-extra-boolean-cast](https://eslint.org/docs/rules/no-extra-boolean-cast) rule.